suitcase/structure.py

```diff
@@ -85,6 +86,11 @@ def unpack_stream(self, stream):
         logic present for dealing with checksum fields.
 
         """
+        def is_substructure(field):
+            return isinstance(field, SubstructureField) or \
+                    isinstance(field, ConditionalField) and \
+                    isinstance(field.field, SubstructureField)

```

Comment:
I can change this.  The main reason for changing is so that user's can add fields that behave similar to `ConditionalField` (wrapping something like a SubstructureField) without having to modify this code (Open/Closed Principle)